Dockerfile

```diff
@@ -3,36 +3,44 @@ FROM golang:1.12.2-alpine3.9 as pingdom_builder
 RUN apk add git
 RUN go get -v github.com/russellcardullo/terraform-provider-pingdom
 
-FROM alpine:3.7
+FROM ruby:2.6.3-alpine
```

Comment:
#34 expects to change the base image to alpine:3.9

This change will make that change harder. Perhaps it would be better to use alpine:3.9 as the base image in this change, and to add ruby using apk.